tests/laser/evm_testsuite/evm_test.py

```diff
@@ -51,20 +52,28 @@ def test_vmtest(test_name: str, pre_condition: dict, action: dict, post_conditio
 
     # Act
     laser_evm.time = datetime.now()
-    execute_message_call(
-        laser_evm,
-        callee_address=action['address'],
-        caller_address=action['caller'],
-        origin_address=action['origin'],
-        code=action['code'][2:],
-        gas=action['gas'],
-        data=binascii.a2b_hex(action['data'][2:]),
-        gas_price=int(action['gasPrice'], 16),
-        value=int(action['value'], 16)
-    )
+    try:
+        execute_message_call(
+            laser_evm,
+            callee_address=action['address'],
+            caller_address=action['caller'],
+            origin_address=action['origin'],
+            code=action['code'][2:],
+            gas=action['gas'],
+            data=binascii.a2b_hex(action['data'][2:]),
+            gas_price=int(action['gasPrice'], 16),
+            value=int(action['value'], 16)
+        )
+    except VmException:
+        return
```

Comment:
Is this correct? This just ignores the test if it fails right?

Reply:
some tests check how errors are dealt so thats why, also i think i should check whether the test expects for the error in the exception catch.

Reply:
changed